src/quo/core.cljs

```diff
@@ -16,6 +16,7 @@
 
 (def text text/text)
 (def header header/header)
+(def header-action header/header-action)
```

Comment:
see other comments about using quo2 components only. Also we generally try to avoid exporting multiple components from the one file, so in general I would prefer they were separated to their own view file. 
However in this case it's not necessary as I think we should avoid using this code completely 👍

Reply:
Removed it.